src/app/utils/field-validation/validators/dateValidator.ts

```diff
@@ -42,10 +42,8 @@ const dateFormatValidator: DateValidator = (response) => {
  */
 const pastOnlyValidator: DateValidator = (response) => {
   // Today takes two possible values - a min (in makeFutureOnlyValidator) and max (here)
-  // Add 14 hours here to account for up to UTC + 14 timezone
-  // This allows validation to pass as long as user is on the correct date (locally)
-  // Even if they are in a different timezone
-  const todayMax = moment().utc().add(14, 'hours').startOf('day')
+  // Dates are converted to use local timezones when loaded by the DateField so no conversion required
+  const todayMax = moment()
```

Comment:
Will this affect server-side validation? E.g. singapore server is on date D, and client is on date D+1

Reply:
actually yea I was a bit confused as to where the validator is actually used haha lemme relook at this

Reply:
ah okay yea I think there will be an issue 😓 thanks for pointing this out!

Reply:
should be fixed in [47cbc25](https://github.com/opengovsg/FormSG/pull/6096/commits/47cbc25f83d6d8f198ee104e9ca1651ae5721452)